src/HydraAuction/Tx/FeeEscrow.hs

```diff
@@ -105,8 +104,8 @@ distributeFee terms = do
           TxOutDatumNone
           ReferenceScriptNone
 
-zipDistributingValue :: Integer -> [PubKeyHash] -> [(PubKeyHash, Value)]
-zipDistributingValue lovelaceAmt delegatePKHs = zip delegatePKHs (lovelaceToValue . Lovelace <$> distributedLovelace)
```

Comment:
Lovelace seems to implement `divMod`, so (un)wrapping is not really necessary.

![image](https://github.com/mlabs-haskell/hydra-auction/assets/300919/72a6e265-8340-4280-8c4f-74879cfb9e1f)